### DISCUSSION TITLE
Multi-app for exchange between a set intersecting 2d planes and a 3d mesh using the same physics kernels

### DISCUSSION THREAD

**Discussion by rpodgorney** [idaholab/moose#16631] [Q&A General] [answered]
MOOSE Users/team--I'm looking for someone to help me put together a demo simulation using a multi-app for heat (and maybe mass) exchange between a set intersecting 2d planes (tris--a 3d fracture network) and a a 3d mesh (tet--a porous media domain)?

**Comment by friedmud** [comment, marked as answer]
Should be able to just do a `MultiAppMeshFunctionTransfer` to transfer both directions.

But - it's going to depend on how you want the lower dimensional domain to feed information up to the higher dimensional domain.

**Reply by rpodgorney** [reply]
thanks @friedmud, that's the kind of info I need---anyone @INL have an interest to play with this?  I have resources but no time!

**Comment by aeslaughter** [comment]
Does @WilkAndy already have something like this?

**Reply by rpodgorney** [reply]
I don't know, hey @WilkAndy have you done this?
